lib/benchee/formatters/unit/duration.ex

```diff
@@ -0,0 +1,59 @@
+defmodule Benchee.Unit.Duration do
+  alias Benchee.Unit.Common
+
+  @behaviour Benchee.Unit
+
+  @microseconds_per_millisecond 1000
+  @milliseconds_per_second 1000
+  @seconds_per_minute 60
+  @minutes_per_hour 60
+  @microseconds_per_second @microseconds_per_millisecond * @milliseconds_per_second
+  @microseconds_per_minute @microseconds_per_second * @seconds_per_minute
+  @microseconds_per_hour @microseconds_per_minute * @minutes_per_hour
+
+  @units %{
+    hour:        %{ magnitude: @microseconds_per_hour, short: "h",  long: "Hours"},
+    minute:      %{ magnitude: @microseconds_per_minute, short: "m",  long: "Minutes"},
+    second:      %{ magnitude: @microseconds_per_second, short: "s",  long: "Seconds"},
+    millisecond: %{ magnitude: @microseconds_per_millisecond, short: "ms", long: "Milliseconds"},
+    microsecond: %{ magnitude: 1, short: "μs", long: "Microseconds"}
```

Comment:
`{}` placement is a bit inconsistent here, the beginning has a space while the end doesn't. Either way is fine with me :)